.github/CHANGELOG.md

```diff
@@ -21,6 +21,12 @@
 * Added `Bargmann` representation (parametrized by Abc). Supports all algebraic operations and CV (exact) inner product.
   [(#296)](https://github.com/XanaduAI/MrMustard/pull/296)
 
+* Added an ArrayAnsatz class. This is used in the Fock representation.
```

Comment:
remove this (we agreed that everything that is in the prototype-tn won't have a changelog entry, until we merge the pieces into develop)